src/frameAndLink.cpp

```diff
@@ -1,23 +1,21 @@
 #include "../src/headers/frameAndLink.h"
 
+/********
+ * LINK *
+ ********/
 
-/****************************
- * LINK                     *
- ****************************/
-
-bool Link::distanceToNextClosestEntry(const Timestamp &tStamp, std::chrono::milliseconds &distanceToCloserEntry) const{
+bool Link::distanceToNextClosestEntry(const Timestamp &tStamp, std::chrono::milliseconds &distanceToCloserEntry) const {
 
     return buf.distanceToNextClosestEntry(tStamp, distanceToCloserEntry);
 }
 
-
 bool Link::addTransformation(const FrameID &srcFrame, StampedTransformation stampedTransformation) {
 
-    // NOTE: assertion just during development
-    // caller should not call the function on a wrong link
+    // NOTE: Assertion just during development.
+    // Caller should not call the function on a wrong link.
     assert((parent->frameID == srcFrame) || ( child->frameID == srcFrame));
 
-    // caller should make sure that the translation is pure
+    // Caller should make sure that the translation is pure.
     assert(stampedTransformation.translation.scalar() == 0.);
```

Comment:
In my understanding a pure quaternion is a quaternion where the scalar value is zero. But as you pointed out, there is actually no need to use a quaternion for the translation part.